app/helpers/application_helper/toolbar/middleware_deployments_center.rb

```diff
@@ -19,4 +20,34 @@ class ApplicationHelper::Toolbar::MiddlewareDeploymentsCenter < ApplicationHelpe
       ]
     ),
   ])
+  button_group('middleware_deployments_operations', [
+    select(
+      :middleware_deployment_deploy_choice,
+      'fa fa-play-circle-o fa-lg',
+      t = N_('Operations'),
+      t,
+      :enabled => false,
+      :onwhen  => "1+",
+      :items => [
+        button(
+          :middleware_deployment_redeploy,
+          'pficon pficon-restart fa-lg',
+          N_('Redeploy these #{ui_lookup(:table=>"middleware_deployments")}'),
+          N_('Redeploy archive'),
+          :url_parms => "main_div",
+          :enabled => false,
+          :onwhen  => "1+",
+          :confirm => N_("Do you want to redeploy these deployments ?")),
+        button(
+          :middleware_deployment_undeploy,
+          'fa fa-eject fa-lg',
+          N_('Undeploy these #{ui_lookup(:table=>"middleware_deployments")}'),
+          N_('Undeploy archive'),
```

Comment:
in the page where multiple deployments are displayed, shouldn't this be in plural? 
Also, the word 'archive' might be misleading here as the entity name in miq is deployment.
So perhaps "undeploy" or "redeploy" are enough. ("Undeploy deployment" perhaps sounds too long)